services/galley/test/integration/API/Teams/LegalHold.hs

```diff
@@ -289,25 +288,35 @@ testGetLegalHoldTeamSettings = do
         -- returns 403 if user is not in team.
         getSettings stranger tid !!! const 403 === statusCode
 
-        -- returns 403 if legalhold disabled for team
-        -- TODO: Uncomment when 'disabled' is the default
-        -- > getSettings owner tid !!! const 403 === statusCode
```

Comment:
Did you add this check back somewhere? It seems to have gone missing.

Reply:
we don't want to return 403 any more, but 200 with `ViewLegalHoldServiceDisabled`.  that's tested in 291-298.  please double-check if that makes sense, not missing anything here is important!